lighthouse-cli/bin.ts

```diff
@@ -44,6 +45,14 @@ interface LighthouseError extends Error {
 
 const cliFlags = getFlags();
 
+if (cliFlags.remoteDevice) {
+  cliFlags.port = defaults(cliFlags.port, 9222);
```

Comment:
Gotcha. Idea behind default was to assign those if supplied. If a user supplies `--remote-device` and `port=8080`, it won't get the override, rather stay at `9222`